src/elasticsearch/donorIndexMapping.json

```diff
@@ -50,6 +50,15 @@
       "alignmentsFailed": {
         "type": "byte"
       },
+      "MutectCompleted": {
```

Comment:
These have different case than the other properties. Let's make them consistent like `mutectCompleted`